blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md

```diff
@@ -0,0 +1,134 @@
+
+_Eva and Georgina met at a GirlsCode MK meetup._
+
+##### Hobbies?
+
+Indoor climbing, singing in a choir, playing table tennis, juggling, watching basketball (NBA!) and spending time outside with my dog.
+
+##### Favourite dish?
+
+Pizza!
+
+##### Morning person or night owl? 
+
+Morning person. I go to bed at 10pm (at the latest!).
+
+##### Which country is at the top of your must-see countries? 
+
+Costa Rica.
+
+##### Favourite programming language? 
+
+Javascript.
+
+##### Favourite productivity hack?
+
+Getting at least 8 hours of sleep. Also coffee. Lots of coffee.
+
+![teamGirlsCodeMK](/img/blog/2018/2018-07-08-4ps-of-girlscode-mk-GirlsCodeMK02.jpg)
+<div class="image-credits"> GirlsCode MK meetups (Photo taken by Eva)</div>
+
+### Coaches and supervisors
+
+On our exciting coding journey, we’ll be joined by a team of enthusiastic, supportive and hopefully (extremely) patient people! <br>
+
+##### Coaches: 
+ 
+Rob, Neil, Kalan and PJ
+ 
+##### Bahmni mentors:
+
+Ivo and Wolf
+
+##### RGSoC supervisor
+
+Ramón 
+
+
+## 2. The Place
+Our office space couldn’t have been more inspiring! We are based at the heart of Bletchley Park home of the codebreakers.
+
+For one of her earlier projects, Georgina built a tribute page for women codebreakers in Bletchley Park ([Codepen link](https://codepen.io/gtl90/pen/opOVvP))
+
+
+![teamGirlsCodeMK](/img/blog/2018/2018-07-08-4ps-of-girlscode-mk-GirlsCodeMK03.jpg)
+<div class="image-credits"> Bletchley Park Mansion - the headquarters for British codebreakers during World War II. (Source: Wikimedia Commons)</div>
+
+<br>
+
+## 3. The Project
+The project will be working on during the summer is Bahmni, a low-resource hospital management tool based on OpenMRS - an open source, medical record system. Bahmni was started in 2012 and is currently being used in 29 countries around the world!
+It’s a live and working app with a huge code base and while it will definitely be challenging, we’re very excited at the opportunity of making Bahmi better and even more effective.
+
+
+![teamGirlsCodeMK](/img/blog/2018/2018-07-08-4ps-of-girlscode-mk-GirlsCodeMK04.png)
+<div class="image-credits"> Bahmni (Source: Bahmni.org)</div>
+
+<br>
+
+## 4. The Plan
+Our primary task is to add a person module to Bahmni. Currently it’s only possible to register people as patients in Bahmni, but we would like to add the functionality of adding persons - for example to register people for pre-screening appointments.
+**Technologies used:** React.js, Node.js, RESTful APIs, PWA
+
+<br>
+
+## You can follow our journey here:
+
+__[@GirlsCodeMK](https://twitter.com/GirlsCodeMK)__
```

Comment:
May be it would be good to write whose twitter accounts these are. You can have your names as a text [linking to twitter link] instead of twitter handles.

Reply:
Thank you Vaishali :)